docker-compose.yml

```diff
@@ -0,0 +1,46 @@
+version: "3"
+services:
+
+  mssql:
+    image: mcr.microsoft.com/mssql/server:${MSSQL_IMAGE_TAG:-2017-latest}
```

Comment:
I was going to ask if it might make sense to pull a newer Docker image, but even MS uses 2017 [here](https://github.com/microsoft/mssql-docker), so I imagine this is fine. Extended support continues until October 2027. Also, I know very little about MS SQL generally. 🙈

(I only mention it given that 2019 and 2022 are also available from Docker Hub, and Nic last edited some of the SQL-related Docker entries for Leaf roughly 3 years ago.)

Reply:
This implies that Leaf has mainly been tested against 2017 and it's known to work, so there would be some testing involved with making sure it plays well with 2019 and/or 2022... worth doing At Some Point, but we have time now.

Reply:
Thanks Kit- that's exactly my sentiment

I'll only add that I tried to minimize changes by matching the version of MSSQL referenced in the official Leaf docs